examples/third-party/haproxy-ingress/10_ingress-default-backend.deploy.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     spec:
       containers:
       - name: ingress-default-backend
-        image: gcr.io/google_containers/defaultbackend:1.0
+        image: docker.io/scylladb/scylla-operator:thirdparty-google_containers-defaultbackend-1.4@sha256:710e7b3ab708ed0fbd3bb4005893bdadf893983441f46c6680ad2ed6f04c261e
```

Comment:
Is this a one time thing? Why isn't this coming from the scylla-operator-images repo?

Reply:
it's one time manual retag because their amd64 and arm64 version live in different repos